src/hal/src/sensors_cf2.c

```diff
@@ -348,9 +351,13 @@ static void sensorsDeviceInit(void)
   // To low DLPF bandwidth might cause instability and decrease agility
   // but it works well for handling vibrations and unbalanced propellers
   // Set output rate (1): 1000 / (1 + 1) = 500Hz
-  mpu6500SetRate(1);
+  mpu6500SetRate(0);
   // Set digital low-pass bandwidth
-  mpu6500SetDLPFMode(MPU6500_DLPF_BW_98);
+  mpu6500SetDLPFMode(MPU6500_DLPF_BW_42);
```

Comment:
This will set the gyro DLPF. Is that what you intended to do or you wanted to change the accel DLPF? I think this setting might slow down the system.